sources/packages/api/src-sql/SFASPartTimeApplications/Create-part-time-sfas-application-records.sql

```diff
@@ -0,0 +1,37 @@
+-- CREATE TABLE sfas_part_time_applications
+CREATE TABLE IF NOT EXISTS sims.sfas_part_time_applications (
+  id VARCHAR(10) PRIMARY KEY,
+  individual_id INT NOT NULL,
+  start_date DATE,
+  end_date DATE,
+  csgp_award NUMERIC(10),
+  sbsd_award NUMERIC(10),
+  -- Audit columns
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+  extracted_at TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE INDEX sfas_part_time_applications_individual_id ON sims.sfas_part_time_applications(individual_id);
+
+-- ## Comments
+COMMENT ON TABLE sims.sfas_part_time_applications IS 'This record contain data related to an Student Application on SAIL(Part Time Application).';
+
+COMMENT ON COLUMN sims.sfas_part_time_applications.id IS 'The unique key/number used in SFAS to identify this application (Sail_extract_data.sail_application_no ).';
```

Comment:
Sail_extract_data or sail_extract_data? Can you please check the other columns also, I feel it's not consistent?

Reply:
![image](https://user-images.githubusercontent.com/77353155/150198388-e8b2f38d-1a49-438e-b407-99ad8db2da4b.png)
its `Sail_extract_data`, you can check the documentation in the wiki tab

Reply:
okay, can you please change the csgp_award and sbsd_award columns comments?

Reply:
I believe that what @guru-aot is saying @ann-aot is that it is not consistently named in the documentation.
You are right about how the doc is but the doc is not consistent 😉